taskcat/_s3_stage.py

```diff
@@ -34,26 +34,10 @@ def stage_in_s3(config):
     """
     bucket_set: set = set()
 
-    # Create the bucket objects first!
     for test in config.tests.values():
         for region in test.regions:
             bucket_set.add(region.s3bucket)
-    # Sync!
     for bucket in bucket_set:
-        try:
-            bucket.create()
-        except Exception as e:
```

Comment:
Why did you remove the exception handling?

Reply:
shouldn't be using `TaskcatException` for catch-all exceptions